projects/gloo/pkg/api/v1/enterprise/options/ai/ai.pb.go

```diff
@@ -300,13 +302,15 @@ var (
 		2: "SUFFIX",
 		3: "CONTAINS",
 		4: "regex",
+		// Duplicate value: 4: "REGEX",
```

Comment:
Interesting option, on the wire, the name doesn't matter but I wonder if we switch the order in the proto file, it will use the "REGEX" for 4 instead of "regex".

Reply:
oh good point, let me try